docs/examples/data/torchvision/data.py

```diff
@@ -0,0 +1,51 @@
+"""Make sure the data is available"""
+import os
+import shutil
+import sys
+import time
+from multiprocessing import Pool
+from pathlib import Path
+
+from torchvision.datasets import INaturalist
+
+
+def link_file(src:str, dest:str):
+    Path(src).symlink_to(dest)
+
+
+def link_files(src:str, dest:str, workers=4):
+    src = Path(src)
+    dest = Path(dest)
+    os.makedirs(dest, exist_ok=True)
+    with Pool(processes=workers) as pool:
+        for path, dnames, fnames in os.walk(str(src)):
+            rel_path = Path(path).relative_to(src)
+            fnames = map(lambda _f: rel_path / _f, fnames)
+            dnames = map(lambda _d: rel_path / _d, dnames)
+            for d in dnames:
+                os.makedirs(str(dest / d), exist_ok=True)
+            pool.starmap(
+                link_file,
+                [(src / _f, dest / _f) for _f in fnames]
+            )
+
+
+if __name__ == "__main__":
+    src = Path(sys.argv[1])
+    dest = Path(sys.argv[2])
+    workers = int(sys.argv[3])
+
+    t = -time.time()
```

Comment:
Let's do this like normal humans do it! :stuck_out_tongue: 
```suggestion
    start_time = time.time()
```